src/jit/importer.cpp

```diff
@@ -7401,7 +7401,7 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
 #endif // defined(DEBUG) || defined(INLINE_DATA)
 
                 // Is it an inline candidate?
-                impMarkInlineCandidate(call, exactContextHnd, callInfo);
+                impMarkInlineCandidate(call, exactContextHnd, exactContextNeedsRuntimeLookup == TRUE, callInfo);
```

Comment:
It could be omitted if `BOOL` and `bool` were the same type, but since they're not, I get warnings about possible "performance issues".